### ISSUE TITLE
[Spec PR Pipeline]:  Enable email notification to SDK breaking change reviewers when the PR is ready for review.

### ISSUE THREAD

**Issue by lirenhe** [Azure/azure-sdk-tools#9532]
**Problem Description**

Today, there are multiple prerequisite for Azure SDK team to review the SDK breaking change including:  the PR should be signed off by ARM team, the API breaking change should be approved already etc.  
It would cause people could not start the review even the PR is marked 'BreakingChange-*-SDK' and as a result, people may miss the SLA for SDK breaking change review.

**Proposed solution**
Enable email notification in the pipeline when a PR is ready for SDK breaking change review. 

cc @raych1   @maririos

**Comment by maririos** [comment]
Not sure I understand all the prerequisites. My understanding is that if a PR is `ARMSignedOff` and automation added an `BreakingChange-<language>-SDK...` then it means the PR is ready to be reviewed by the SDK team.
If that is true, a filter per language like this should work to monitor the work: https://github.com/Azure/azure-rest-api-specs/pulls?q=is%3Aopen+is%3Apr+label%3AARMSignedOff+label%3ABreakingChange-Go-Sdk

**Comment by lirenhe** [comment]
There are some PRs that do not require ARMSignedOff for the configuration/directive change but they would also cause SDK breaking change.
And some PRs that also requires API breaking change approved first before starting SDK breaking change review.

We have the SLA of 2 business days but it is quite easy to miss considering the different combination of scenarios by manual check.
I think we have similar notification for ARMView, so can we quickly leverage that for SDK breaking change review?

**Comment by raych1** [comment]
Right, the condition to determine a PR is ready for SDK breaking change review is indeed complex, and constructing a query URL to filter out these PRs accurately is not feasible. 

As for the approach, I propose adding a new label, such as `WaitForSdkBreakingChangeReview` to indicate that a spec PR is ready for this review. This would allow reviewers to easily monitor this label and identify such PRs.

**Comment by kurtzeborn** [comment]
I feel like we queries with the labels gets us 90% of the way to what's being asked for here.  If there are a few false positives in the query results, I think whoever's viewing/reviewing them (someone on our team) can deal with the occasional false positive that comes up.  We don't need to attempt to engineer a perfect solution when the audience is us and our team is educated on the potential of false positives.

@lirenhe, answering your question, the ARM team does not have a similar notification.  They just use queries with a similar understanding to what I described above.

If we do the 90% solution I described above and, after a few months, it indeed is super cumbersome to have false positives in the query results, I'm happy to revisit this ask.
